conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -29,7 +29,13 @@ dependencies:
 - statsmodels
 - protobuf >=3.4.1,<4.0.0
 - doxygen
+- sphinx
```

Comment:
should we fix a version to match ops build or do they use the latest as well? (I noticed sphinx versions can be a little picky)